discord_music_bot/commands/queue.py

```diff
@@ -69,8 +69,18 @@ async def clear_queue(self, ctx: discord.ApplicationContext) -> None:
     )
     async def play_next(self, ctx: discord.ApplicationContext) -> None:
         player = await get_current_player(ctx)
-
-        # .stop() will trigger on_wavelink_track_end event
-        # that automatically starts playing next track
-        await player.stop()
-        await ctx.respond("Skipping current track...")
+        skipped_item_title = player.track.title
+
+        if not player.queue.is_empty:
+            current_item = player.queue.get()
+
+            await player.play(current_item)
+            await ctx.respond(
+                f"Skipping: **{skipped_item_title}**"
+                f"\nCurrent: **{current_item.title}**"
```

Comment:
Change the message, so it looks like this one:

https://github.com/andrewyazura/discord-music-bot/blob/2e7c6d1ada2e190319f2c4159331aceb4b799e0b/discord_music_bot/commands/streaming.py#L125

Reply:
And consider splitting the message in two, using `ctx.send` for the second part